dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -111,10 +111,17 @@ def cleanRow(row, fileType, validationInterface):
         for key in row.keys():
             field_type = validationInterface.getColumn(key, fileType).field_type.name
             value = row[key]
-            if value is not None and field_type in ["INT", "DECIMAL", "LONG"]:
-                tempValue = value.replace(",","")
-                if FieldCleaner.isNumeric(tempValue):
-                    row[key] = tempValue
+            if value is not None:
+                # Remove extra whitespace
+                value = value.strip()
+                if field_type in ["INT", "DECIMAL", "LONG"]:
+                    tempValue = value.replace(",","")
```

Comment:
The comma removal for numeric values is not new (it's also in the deleted section here), I think this was the way we wound up ignoring commas, it had to be done before validation so the values could pass the type check.